service/mapper/transitions.go

```diff
@@ -269,9 +269,116 @@ func (t *Transitions) ResumeIndexing(s *State) error {
 	// then proceed with the first non-indexed block and index values
 	s.height = last + 1
 
-	// At this point, we should be able to start indexing the chain data for
+	// At this point, we should be able to start indexing the register data for
 	// the next height.
-	s.status = StatusIndex
+	s.status = StatusUpdate
```

Comment:
can we document somewhere that the state transition order?

Reply:
documented above and in `service/mapper/status.go`